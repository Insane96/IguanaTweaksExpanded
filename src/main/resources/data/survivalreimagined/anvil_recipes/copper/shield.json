{
	"item_to_repair": {
		"id": "shieldsplus:copper_shield"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:copper_ingot"
			},
			"amount": 4
		}
	]
}