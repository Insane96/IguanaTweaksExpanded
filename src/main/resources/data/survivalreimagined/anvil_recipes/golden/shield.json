{
	"item_to_repair": {
		"id": "shieldsplus:golden_shield"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:golden_ingot"
			},
			"amount": 4
		},
		{
			"repair_material": {
				"id": "minecraft:golden_nugget"
			},
			"amount": 36
		}
	]
}